packages/slice-machine/components/ListItem/index.tsx

```diff
@@ -12,27 +12,49 @@ import { AiOutlineEdit } from "react-icons/ai";
 import { BsThreeDotsVertical } from "react-icons/bs";
 
 import { FaBars } from "react-icons/fa";
+import { Widget } from "@lib/models/common/widgets/Widget";
+import { Field } from "@lib/models/common/CustomType/fields";
+import { AnyObjectSchema } from "yup";
```

Comment:
What's this do?

Reply:
I took it from the definition of a widget, i think it's just a schema to validate any object, it must have been something prior to io-ts.
It will be resolved once the Field / Widget types are all put in common.